deepchecks/checks/performance/roc_report.py

```diff
@@ -126,22 +126,22 @@ def add_condition_auc_not_less_than(self, min_auc: float = 0.7):
 
         """
         def condition(result: Dict) -> ConditionResult:
-            failed_classes = []
+            failed_classes = {}
             for item in result.items():
                 class_name, score = item
                 if score < min_auc:
-                    failed_classes.append(f'class {class_name}: {format_number(score)}')
+                    failed_classes[class_name] = format_number(score)
             if failed_classes:
                 return ConditionResult(False,
-                                       f'The scores that are less than the allowed AUC are: {failed_classes}')
+                                       f'Found classes with subceeding AUC: {failed_classes}')
```

Comment:
what was the low auc?

Reply:
I don't understand the question

Reply:
does this also print what was the failing AUC?

Reply:
yes